src/convertToObject.js

```diff
@@ -13,6 +13,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arrOfString = sourceString
```

Comment:
CheckList
2. [CODE STYLE] - Avoid variable namings like `array` , `object` `arr`, `obj`, `result`. It will decrease code readability